build.gradle

```diff
@@ -52,15 +54,19 @@ buildscript {
 }
 
 def FG3 = !fabric && mcVersion >= 11400
+def FG1_2 = !fabric && mcVersion <= 10800
 def FABRIC = fabric
 
 if (FABRIC) {
     apply plugin: 'fabric-loom'
+} else if (FG1_2) {
```

Comment:
I would say this after the `} else if (FG3) { 
    apply plugin: 'net.minecraftforge.gradle'` as above It's after it.